### DISCUSSION TITLE
CVE-2024-1086 vulnerability, is my cluster at risk?

### DISCUSSION THREAD

**Discussion by clbarrett** [Azure/AKS#4343] [General]
Just upgraded my linux node pools to AKSUbuntu-2204gen2containerd-202405.20.0.  Is there any risks to my AKS nodes / clusters due to this vulnerability?

https://arstechnica.com/security/2024/05/federal-agency-warns-critical-linux-vulnerability-being-actively-exploited/

**Comment by tspearconquest** [comment]
https://ubuntu.com/security/CVE-2024-1086

**Reply by tspearconquest** [reply]
`kubectl get nodes -o wide`

Look at the kernel version. If it's 5.15.0-1059.67 or higher, you are patched.
